docs/references/api/preferences.rst

```diff
@@ -264,3 +264,31 @@ To illustrate the use of this preference, consider the following scenario where
       "details": "The query affects 14 rows",
       "hint": null
   }
+
+With `RPC <functions>`_, the preference is honored completely on the basis of the number of rows returned in the result set of the function. This can be useful for complex mutation queries using [data-modifying statements](https://www.postgresql.org/docs/current/queries-with.html#QUERIES-WITH-MODIFYING). A simple example:
```

Comment:
```suggestion
With `RPC <functions>`_, the preference is honored completely on the basis of the number of rows returned in the result set of the function. This can be useful for complex mutation queries using `data-modifying statements <https://www.postgresql.org/docs/current/queries-with.html#QUERIES-WITH-MODIFYING>`_. A simple example:
```